source/administration/telemetry.rst

```diff
@@ -96,20 +96,20 @@ Trace events
   Includes duration on how long the action took place like startup, team/channel switch, posts loading/update and channel drawer open/close. The naming convention is interpreted as ``[start observation]:[end observation]``, e.g. ``start:overall`` as from app start until fully rendered or ``post_list:thread`` as on press of post at post list until thread is opened.
   Complete list of trace events are the following:
 
-  - start:overall
-  - start:process_packages
-  - start:content_appeared
-  - start:select_server_screen
-  - start:channel_screen
-  - team:switch
-  - channel:loading
-  - channel:switch_loaded
-  - channel:switch_initial
-  - channel:close_drawer
-  - channel:open_drawer
-  - posts:loading
-  - post_list:thread
-  - post_list:permalink
+  - *start:overall*: Initial start (cold or reload) until the last post of a channel is loaded.
```

Comment:
End users do not really have a way to reload, right?

Reply:
@enahum what change are you proposing here?

Reply:
```suggestion
  - *start:overall*: Application cold start until the last post of a channel is loaded.
```